sdl_android_lib/src/com/smartdevicelink/transport/USBTransport.java

```diff
@@ -487,7 +491,7 @@ private void initializeAccessory() {
      * @param accessory Accessory to check
      * @return true if the accessory is right
      */
-    private boolean isAccessorySupported(UsbAccessory accessory) {
+    public static boolean isAccessorySupported(UsbAccessory accessory) {

```

Comment:
The app has ability to create a transport config object with a UsbAccessory object (as seen in the link below), however their is no publicly available method for the app to use to check if the accessory is supported.  

https://github.com/smartdevicelink/sdl_android/blob/develop/sdl_android_lib/src/com/smartdevicelink/transport/USBTransportConfig.java#L29